group-recommendations.html

```diff
@@ -0,0 +1,104 @@
+﻿<!DOCTYPE html>
+<html>
+
+<head>
+    <!--Put  Tab Title Here-->
+    <title>Group Recommendations</title>
+    <!--Set General Style Settings-->
+    <style>
+        .navbar {
+            padding-top: 8px !important;
+            padding-bottom: 8px !important;
+        }
+
+        .content {
+            background-color: #F9F9F9;
+            width: 60%;
+            padding-top: 10px;
+            padding-bottom: 20px;
+            padding-left: 10px;
+            padding-right: 10px;
+            margin-right: 20%;
+            margin-left: 20%;
+            border-radius: 17px;
+        }
+
+        tr {
+            border: none !important;
+        }
+
+        td {
+            border: none !important;
+        }
+
+        .input {
+            height: 30px !important;
+        }
+    </style>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
+    <script src="group-recommendations.js"></script>
+
+</head>
+
+<body onload="groupMovies()">
```

Comment:
We will have to pair program the HTML later on when I incorporate the watched button. Greate start on this though!